webapp/views.py

```diff
@@ -60,6 +60,7 @@ def advantage():
     enterprise_contracts = []
     entitlements = {}
     openid = flask.session.get("openid")
+    headers = {"Cache-Control": "no-store"}
```

Comment:
It occurred to me that `private` is probably more accurate.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Cacheability

Reply:
updated